.github/header-checker-lint.json

```diff
@@ -0,0 +1,5 @@
+{
+    "sourceFileExtensions": ["js", "py"],
+    "allowedLicenses": ["Apache-2.0"],
+    "ignoreFiles": ["**/node_modules/*"]
```

Comment:
I think it should be `.github/header-checker-lint.yml`
And yeah, you need to convert to yaml

Reply:
https://github.com/googleapis/repo-automation-bots/issues/897